network-api/networkapi/wagtailpages/models.py

```diff
@@ -64,6 +66,21 @@ def meta_title(self):
 
     show_in_menus_default = True
 
+    def get_meta_description(self):
+        if self.search_description:
+            return self.search_description
+        return (
+            "Mozilla is fueling a movement for a healthy Internet. "
+            "See how a network of dedicated leaders and activists "
+            "are working for a better Internet for all, connecting "
+            "us to each other and to a better future."
+        )
+
+    def get_meta_image(self):
+        if self.search_image:
+            return self.search_image
+        return Image.objects.create(title='Share Image', file='images/og.jpg')
```

Comment:
It seems this doesn't quite do what I expected it (needed it) to do.

I just need an object variable that looks like the image object, and contains the new source.

What is actually happening is every time this is run, it makes a copy of the image and puts it in /network-api/media/images

Reply:
I think the default image might be a nice to have, but otherwise I'm not sure how to fix yet.

Reply:
I think default share images might be too much to ask for. For now I'll remove this as it's causing all sorts of problems.